### PR TITLE
add liquidation tests in integration

### PR DESCRIPTION
**Pull request by dbeal-eth**
still needs to be filled out with a couple more checks but overall
the process is there.

wondering if a little bit more code should be added to the helpers for
rates? let me know if this code is clean enough or not.

### CODE REVIEW

**Review by barrasso — APPROVED** [review]
LGTM, just need to run linter

File: test/integration/behaviors/liquidations.behavior.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+				assert.equal(await Liquidations.isOpenForLiquidation(someUser.address), false);
+			});
+
+			it('deadline hasn not passed yet', async () => {
+				assert.equal(await Liquidations.isLiquidationDeadlinePassed(someUser.address), false);
+			});
+
+			describe('getting liquidated', () => {
+				before('otherUser calls liquidateDelinquentAccount', async () => {
+					await skipLiquidationDelay({ ctx });
```
Comment (justinjmoses) [inline test/integration/behaviors/liquidations.behavior.js:92]: I'd put this wait in a `describe` block so it reads `when the liquidation delay passes`

File: test/integration/behaviors/liquidations.behavior.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+					await Synthetix.connect(otherUser).liquidateDelinquentAccount(
+						someUser.address,
+						ethers.utils.parseEther('100')
+					);
+				});
+
+				it('is liquidated', async () => {
+					// = sUSD liquidated / SNX Price * 1.1
+					assert.equal(await Synthetix.balanceOf(someUser.address), '62068965517241379313');
+				});
```
Comment (justinjmoses) [inline test/integration/behaviors/liquidations.behavior.js:103]: can you also check sUSD has been burned from `otherUser`?

**Review by justinjmoses — CHANGES_REQUESTED** [review]
Just a few small things, but 100% the right direction.

Can you also fix the `fork-tests` - you can use the `updateExchangeRatesIfNeeded` helper

File: test/integration/behaviors/liquidations.behavior.js [hunk 3]
```diff
@@ -87,19 +87,37 @@ function itCanLiquidate({ ctx }) {
...
+					});
+
+					it('is liquidated', async () => {
+						// = sUSD liquidated / SNX Price * 1.1
+						assert.notEqual(await Synthetix.balanceOf(someUser.address), '100000000000000000000');
+					});
+
+					it('deducts sUSD debt from the liquidated', async () => {
+						assert.equal(
+							await Synthetix.debtBalanceOf(otherUser.address, toBytes32('sUSD')),
```
Comment (justinjmoses) [inline test/integration/behaviors/liquidations.behavior.js:117]: it burns sUSD from `otherUser` right? the debt balance of `someUser` should go down and the sUSD balance should go down for `otherUser`

File: test/integration/behaviors/liquidations.behavior.js (hunk 1 above)
Comment (barrasso) [reply test/integration/behaviors/liquidations.behavior.js:92]: https://github.com/Synthetixio/synthetix/pull/1625/commits/7387c9ec0f7b803bab7267f2b8a51a00cbaf6281

File: test/integration/behaviors/liquidations.behavior.js (hunk 3 above)
Comment (barrasso) [reply test/integration/behaviors/liquidations.behavior.js]: https://github.com/Synthetixio/synthetix/commit/11dad8196b1b8a61bc8c751ebffbb8fb2c5c9d8d

File: test/integration/behaviors/liquidations.behavior.js (hunk 2 above)
Comment (barrasso) [reply test/integration/behaviors/liquidations.behavior.js:117]: https://github.com/Synthetixio/synthetix/commit/11dad8196b1b8a61bc8c751ebffbb8fb2c5c9d8d

File: test/integration/behaviors/liquidations.behavior.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+						await Synthetix.connect(otherUser).liquidateDelinquentAccount(
+							someUser.address,
+							ethers.utils.parseEther('100')
+						);
+					});
+
+					it('is liquidated', async () => {
+						// = sUSD liquidated / SNX Price * 1.1
+						assert.notEqual(await Synthetix.balanceOf(someUser.address), '100000000000000000000');
```
Comment (justinjmoses) [inline test/integration/behaviors/liquidations.behavior.js:104]: [np] I'd at least use `assert.bnLt` rather than `notEqual`

**Review by justinjmoses — APPROVED** [review]
Approved! Just one minor nitpick

File: test/integration/behaviors/liquidations.behavior.js (hunk 4 above)
Comment (barrasso) [reply test/integration/behaviors/liquidations.behavior.js]: https://github.com/Synthetixio/synthetix/pull/1625/commits/658db9280e3f0ea6d82bdd5398f2edb80b10451f

**Pull request merged**